ch5/immutable=params.zig

```diff
@@ -0,0 +1,13 @@
+const std = @import("std");
```

Comment:
Just to avoid any possible errors (and maybe it's just a typo) but the file name would be better without the equal sign, `immutable_params.zig` is better

Reply:
I was a typo 😅